### DISCUSSION TITLE
blog/distributed-tracing-with-skywalking-in-istio/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#332] [Announcements]
# blog/distributed-tracing-with-skywalking-in-istio/

这篇文章将介绍一些关于分布式追踪的基础知识以及如何在 Istio 中使用分布式追踪。

https://jimmysong.io/blog/distributed-tracing-with-skywalking-in-istio/

**Comment by dhl5168** [comment]
按照此文章部署了bookinfo，但是在skywalking里面一直显示no data，这是什么原因？

**Reply by rootsongjc** [reply]
你有给 productpage 服务发送一些流量试一试吗？

**Comment by rootsongjc** [comment]
https://github.com/istio/istio/discussions/48325 报告了错误，经检查发现博客中在说明安装 Istio 的地方有一处命令错误，现已修正。经本人在 Kind 中实际测验，本博客同样适用于最新的 Istio 1.20.1 版本。

**Reply by rootsongjc** [reply]
另外不建议使用 Istio Operator 方式安装 Istio，即不要执行 `istioctl operator init` 命令，而是直接使用 `istioctl` 命令安装 Istio。
